package.json

```diff
@@ -4,14 +4,18 @@
   "main": "src/index.js",
   "repository": "tobiaslins/avatar",
   "scripts": {
-    "start": "micro src/index.js"
+    "start": "NODE_ENV=production micro src/index.js",
```

Comment:
@timolins If you have the `main` field specified, you don't need to pass the path to micro or micro-dev. 😊